src/components/TabBar.js

```diff
@@ -0,0 +1,186 @@
+import _ from 'lodash/fp'
```

Comment:
We generally only name a file with a capital letter when it has a default export of a component with the same name; since there are two exports here, would you consider either merging them to have a prop that says which version someone wants, and then making it a default export, or changing the name to i.e. `tabBars.js`?

Reply:
OOHH! 💡  I was wondering why there was a difference in capitalization!  Thanks for explaining. I think I'll just rename the file. :)